public/static/js/build.js

```diff
@@ -80,48 +80,56 @@ mciModule.controller('BuildViewController', function($scope, $http, $timeout, $r
     {
       trigger: "outcome",
       resource_type: "BUILD",
-      label: "this build finishes"
+      label: "this build finishes",
+      regex_selectors: buildRegexSelectors(),
```

Comment:
But this is in the context of a single build. It definitely makes sense for the same trigger at a project level, but not here

Reply:
Yep, sorry i cleaned up the task level stuff but the not the build level stuff. Done, but I've left some unused functions around because we'll be using them in another ticket in this epic